NTupleAnalyzer_nanoAOD.py

```diff
@@ -132,6 +142,7 @@
 _kinematicvariables += ['M_jj']
 _kinematicvariables += ['DR_muon1muon2','DPhi_muon1met','DPhi_jet1met','DPhi_jet2met']
 _kinematicvariables += ['DR_muon1jet1','DR_muon1jet2','DR_muon2jet1','DR_muon2jet2']
+_kinematicvariables += ['DR_muon1muon2jet1']
```

Comment:
I don't like this name, all the others, e.g. DR_muon1muon2 means the DR between muon1 and muon2, but this one means DR between the 2-muon object and jet1, so maybe let's call it DR_dimujet1 ? do you have a better idea?

Reply:
Yeah I agree with you, there. I went with DR_dimuonjet1 for the branch name, and _DRdimujet1 for the local variable.